package.json

```diff
@@ -152,6 +152,7 @@
     "focus-trap-react": "^10.0.2",
     "graphql": "^16.5.0",
     "graphql-tag": "^2.12.6",
+    "http-error-classes": "^1.0.1",
```

Comment:
Please check the status of your merges from other branches.

This package has been pulled out from #356. It is still present here.